Makefile.js

```diff
@@ -670,168 +668,6 @@ target.webpack = function(mode = "none") {
     exec(`${getBinFile("webpack")} --mode=${mode} --output-path=${BUILD_DIR}`);
 };
 
-target.checkRuleFiles = function() {
```

Comment:
We can remove checks related to `tools/rule-types.json`, but all other checks should remain.